basics/src/main/kotlin/de/thermondo/basics/Basics06.kt

```diff
@@ -0,0 +1,11 @@
+package de.thermondo.basics
+
+/**
+ * Create a class named Basics6Solution
+ *
+ * Define member variables:
+ * @param name: String
+ * @param age: Int
```

Comment:
Since we already learned how to create classes in Basic05, the focus should be on the constructor, and passing parameters to the class.

```suggestion
 * You can pass parameters to classes via the class constructor by first giving the parameter name
 * and then the type, similar to how we create functions with parameters.
 *
 * Let's create a class called Basics06, that accepts the following parameters:
 * 
 * @param name: String
 * @param age: Int
```